internal/ui/assets/css/styles.css

```diff
@@ -122,9 +122,37 @@ header.cdc-header .header-search {
   padding-top: 6px;
 }
 
-.vh {
-  height: 50vh;
-  background-color: orange;
+.usa-accordion__button {
+  margin-bottom: 12px;
+}
+
+.usa-button {
+  height: 3rem;
```

Comment:
can add `max-width: 20rem` to make this match the size of the dropdown on the left side